lib/instrumentation/modules/express.js

```diff
@@ -12,12 +12,45 @@ module.exports = function (express, agent, version) {
     return express
   }
 
+  // express 5 moves the router methods onto a prototype
```

Comment:
Nice that you're thinking of Express 5 already 👍

Reply:
I haven't fully tested express 5, but that was an easy part to fix.